### PR TITLE
Release 1.0

### LINKED ISSUE #8: Update offchain dependencies 0.9.0
As a last task before releasing a new version we must update the project's dependencies and then ensure that all tests pass. 

### Subtasks

- [ ] Run `cargo update`.
- [ ] Run the tests.

### PR DESCRIPTION
**Pull request by gligneul**
Close #8

### CODE REVIEW

**Comment by gligneul** [comment]
Before releasing, we still need to:
- Update the emulator SDK and server-manager references
- Update the onchain dependencies
- Bump the onchain npm libraries versions
- Generate the change log

**Review by guidanoli — COMMENTED** [review]
I am in favor of stylizing code-like names with backticks.
As a bonus, this removes the need to escape some underscores.

- `InvalidClaimIndex`
- `rollups-cli`
- `RPC_URL`
- `InputBox`
- `InputRelay`
- `inboxInputIndex`
- `inputIndex`
- `CREATE2`
- `OutputValidityProof`
- `host-server-manager`
- `host-runner`
- `rollups-deployments`
- `@cartesi/util`
- `ConsensusCreated`
- `Authority`
- `IInputBox`
- `grpc_health_check`
- `polygon_mumbai`
- `bsc_testnet`
- `iotex_testnet`

Also, I prefer on-chain and off-chain over "onchain" and "offchain".

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,55 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Added Prometheus metrics to the `dispatcher` service
+- Added Redis TLS support to the node
+- Added Redis cluster support to the node
+- Added AWS KMS to sign node transactions
+- Added Web Identity support to the AWS KMS signer
+- Added README for the node
+
+### Changed
+
+- Started using custom errors integration in contracts
```
Comment (guidanoli) [inline CHANGELOG.md:29]:
```suggestion
- Started using custom errors in contracts
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,55 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Changed portals and relays inheritance to `InputRelay`
```
Comment (guidanoli) [inline CHANGELOG.md:30]:
```suggestion
- Made portals and relays inherit `InputRelay`
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,55 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Renamed `inboxInputIndex` to `inputIndex` in contracts
+- Deployed contracts deterministically with `CREATE2` factory
+- Improve proof generation system for on-chain tests
```
Comment (guidanoli) [inline CHANGELOG.md:33]:
```suggestion
- Improved proof generation system for on-chain tests
```

**Review by tuler — APPROVED** [review]
LGTM

**Review by guidanoli — APPROVED** [review]

**Pull request merged**